### PR TITLE
Drop jquery.dotdotdot for a lighter css-only solution

### PR DESCRIPTION
**Pull request by noirbizarre**
This PR drop `jquery-dotdotdot` for a lighter css-only solution that should improve memory consumption

Line clamping, webkit version:
screenshot-www data dev 2017-04-06 13-25-35

Firefox/IE version (only difference, no `...` ellipsis :
screenshot-www data dev-2017-04-06-14-12-21

### CODE REVIEW

**Comment by davidbgk** [comment]
I'm confused with provided screenshots 🤔

**Comment by noirbizarre** [comment]
Confused ? Why ? Because the Firefox one is more blurry ?
The only part that matter is the title: ellipsis on 2 lines. The only difference: no `...` on firefox.

**Comment by davidbgk** [comment]
Alright thanks, the example confused me because it looks spam-y :)

Anyway, I don't get why sometimes you use `ellipsis`, sometimes `clamp-2` and sometimes nothing to replace the previous `ellipsis-dot`

**Comment by noirbizarre** [comment]
- `ellipsis`: standard css-only truncation on 1 line
- `clamp-2`: trick to perform a 2 line truncation:
   - webkit: use `-webkit-box`/`-webkit-line-clamp` trick
   - others: simply overflow hidden (but I just found a possible amelioration I'll try with `:atfer` pseudo element to add the ellipsis symbol)
On some element, the `ellipsis-dot` wasn't doing anythin anymore so I simply removed it.

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,6 +22,9 @@
 - bleach and html5lib have been updated leading to more secure html/markdown cleanup
   and (better performances)[http://bluesock.org/~willkg/blog/dev/bleach_2_0.html]
   [#838](https://github.com/opendatateam/udata/pull/838)
+- drop jquery.dotdotdot for a lighter css-only solution (less memomry consumption)
```
Comment (davidbgk) [inline CHANGELOG.md]: Typo: `memory`

File: udata/templates/organization/search-result.html [hunk 2]
```diff
@@ -8,8 +8,8 @@
                 width="70" height="70">
         </div>
         {{ badge_if_certified(organization) }}
-        <div class="result-body ellipsis-dot">
-            <h4 class="result-title">{{ organization.name }}</h4>
+        <div class="result-body ellipsis">
```
Comment (davidbgk) [inline udata/templates/organization/search-result.html]: Why not `ellipsis` on `h4` below and `clamp-2` on the following `div` following the same pattern as others?

Comment (noirbizarre) [inline udata/templates/organization/search-result.html]: mistake on this one, it should be on the h4

**Pull request merged**